tools-local/tasks/installer.tasks/installer.tasks.csproj

```diff
@@ -20,6 +20,14 @@
     <PackageReference Include="System.Reflection.Metadata" Version="1.7.0" />
   </ItemGroup>
 
+  <!--
+    Restore the packaging package here to use it during the repo restore phase in 
+    harvestPackages.targets which is hooked into Tools.props.
```

Comment:
Interesting hack.  I don't suppose there was a cleaner way to get this restored earlier?

Reply:
Unfortunately not unless we bring the necessary task into the repo or sequence the harvesting package download out of the repo restore phase.

Reply:
I see.  Do our Proj files support restore?  If so we could have repo restore download the package and packages.proj do the PackageDownloads.

Reply:
That's a great idea. Yeah *proj files are restorable by default since a nuget update (I believe last year).